src/parser/shaman/enhancement/modules/azerite/RoilingStorm.tsx

```diff
@@ -45,15 +47,31 @@ class RoilingStorm extends Analyzer {
         .spell(STORMSTRIKE_CAST_SPELLS),
       this.onStrikeCast,
     );
+
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER)
+        .spell(STORMSTRIKE_DAMAGE_SPELLS),
+      this.onStrikeDamage,
+    );
   }
 
-  // We do this on cast since the damage is split amongst the two hits.
   onStrikeCast(event: CastEvent) {
+    this.lastAttackPower = event.attackPower || 0;
+  }
+
+  onStrikeDamage(event: DamageEvent) {
     if (!this.selectedCombatant.hasBuff(SPELLS.STORMBRINGER_BUFF.id)) {
       return;
     }
 
-    this.damageGained += this.bonusDamage;
+    let scale = 2/3;
+
+    if (event.ability.guid === SPELLS.STORMSTRIKE_DAMAGE_OFFHAND || event.ability.guid === SPELLS.WINDSTRIKE_DAMAGE_OFFHAND) {
```

Comment:
this wont actually match.

You would need to do `event.ability.guid === SPELLS.STORMSTRIKE_DAMAGE_OFFHAND.id` to get this to work.

Sidenote, this can probably all be replaced by a simple ternary
```
const spellId = event.ability.guid;
const scale = (spellId === SPELLS.STORMSTRIKE_DAMAGE_OFFHAND.id || spellId === SPELLS.WINDSTRIKE_DAMAGE_OFFHAND.id) ? 1/3 : 2/3;
```

Your way is fine too, so dont necessarily need to change it, just seems simpler imo